src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Represents a task entered by user.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Creates task.
+     * @param description Task description.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Checks whether the task is done.
+     * @return Unicode character.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
```

Comment:
I assume these two values are the 'tick' and 'cross'. Conside using a named constant as part of keeping code quality.